.gitignore

```
@@ -10,8 +10,10 @@
 build/
 install/
 log/
+src/
```

Will you make this local?
I guess `src/` might be added to somewhere in the future.

```suggestion
/src/
```

---

I've updated this in https://github.com/autowarefoundation/autoware/pull/121/commits/5cd80567f7878a5eca56b19d66f97ae58fb320ee.
@xmfcx Is it okay for you?